sdk/schemaregistry/Azure.Data.SchemaRegistry/src/ContentTypeInternal.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Data.SchemaRegistry
+{
+    /// <summary>
+    /// Properties for a SchemaRegistry schema.
+    /// </summary>
+    [CodeGenModel("SchemaFormat")]
```

Comment:
I'm not sure that we need this customization. Could we just update the swagger to have the x-ms-enum name that we want - probably just ContentType would be fine.